cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/stream/FileStreamAdmin.java

```diff
@@ -747,4 +768,25 @@ private void alterExploreStream(Id.Stream stream, boolean enable, @Nullable Form
   private void publishAudit(Id.Stream stream, AuditType auditType) {
     AuditPublishers.publishAudit(auditPublisher, stream, auditType, AuditPayload.EMPTY_PAYLOAD);
   }
+
+  private <T extends EntityId> void ensureAccess(T entityId) throws Exception {
+    ensureAccess(entityId, ImmutableSet.<Action>of());
```

Comment:
what's it mean to ensure access with an empty set of actions? Semantically speaking, wouldn't that always pass?